ui/components/config/mcp-clients-lists.tsx

```
@@ -1,179 +1,183 @@
+    loadClients()
+  }
 
-	const getConnectionDisplay = (client: MCPClient) => {
-		if (client.config.connection_type === "stdio") {
-			return client.config.stdio_config?.command + " " + client.config.stdio_config?.args.join(" ") || "STDIO";
-		}
-		return client.config.connection_string || `${client.config.connection_type.toUpperCase()}`;
-	};
+  const getConnectionDisplay = (client: MCPClient) => {
+    if (client.config.connection_type === 'stdio') {
+      return client.config.stdio_config?.command + ' ' + client.config.stdio_config?.args.join(' ') || 'STDIO'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)